apps/hash-api/README.md

```diff
@@ -7,11 +7,18 @@ The HASH Backend API service is configured using the following environment varia
 - `NODE_ENV`: ("development" or "production") the runtime environment. Controls
   default logging levels and output formatting.
 - `PORT`: the port number the API will listen on.
-- `AWS_REGION`: the AWS region to use for the Simple Email Service (SES) provider.
-- S3 file uploads:
+- `AWS_REGION`: the AWS region to use for the Simple Email Service (SES) provider
```

Comment:
@CiaranMn Shall we rename this `AWS_SES_REGION`? It seems a bit weird with `AWS_S3_REGION` to have this just named `AWS_REGION`.

Reply:
It isn't only SES-related, because the S3 config will default to `AWS_REGION` if `AWS_S3_REGION` is undefined

Reply:
We could do with clearing up these READMEs because it seems pointless to explain all these variables in both the API readme and the higher-level one